mocks/wiremock/mappings/prd/organisationsUsersWithRoles.json

```diff
@@ -0,0 +1,21 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPath": "/refdata/external/v1/organisations/users",
+    "queryParameters": {
+      "status": {
+        "equalTo": "active"
+      },
+      "returnRoles": {
+        "equalTo": "true"
+      }
+    }
+  },
+  "response": {
+    "status": 200,
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "bodyFileName": "mockResponse.json"
```

Comment:
minor but do you think it would be good to update the name of this mockResponse.json file? If yes do you think that the last echo in `bin/add-users.sh` from
```sh
echo "\nGenerating local user mappings"
```
to
```sh
printf "\nGenerating local wiremock mappings\n"
```
as echo does not expand escape sequence characters so it just prints
```
\nGenerating local user mappings
```

Reply:
absolutely, done